central/alert/service/service_impl.go

```diff
@@ -103,7 +104,7 @@ func (s *serviceImpl) GetAlert(ctx context.Context, request *v1.ResourceByID) (*
 		return nil, err
 	}
 	if !exists {
-		return nil, errors.Wrapf(errorhelpers.ErrNotFound, "alert with id '%s' does not exist", request.GetId())
+		return nil, errors.Wrapf(errox.NotFound, "alert with id '%s' does not exist", request.GetId())
```

Comment:
Would it be possible to replace such cases with
```suggestion
		return nil, errox.NotFound.Newf("alert with id '%s' does not exist", request.GetId())
```
?
That would remove the useless "not found" from the resulting error message.

Reply:
@0x656b694d Thank you for the suggestion. We want to make changes in several PRs so that it's easier to review them. This PR is the simple removal of `pkg/errorhelpers/errors.go`. Next will be removing helper (wrapper) functions for CausedBy. After that, we can start with further polishing. We can decide if we want to keep the message prefix or remove it.